.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+inherit_from:
+  - https://raw.githubusercontent.com/cookpad/guides/master/.rubocop.yml
```

Comment:
Using Rubocop to analyze this gem's code sounds good, but I'm not sure using `cookpad` guidelines for a public gem is the way to go?

What would you think of using the [Ruby community guidelines](https://github.com/rubocop-hq/ruby-style-guide)?

Reply:
Used this because balvig's guidelines matched cookpad's and without this my linter was all over the place 😂 will change it 👍

Reply:
Does rubocop use  Ruby community guidelines if no settings are specified?